vendor/qtidataservices_app.te

```diff
@@ -0,0 +1,20 @@
+type qtidataservices_app, domain;
+app_domain(qtidataservices_app)
+net_domain(qtidataservices_app)
+
+get_prop(qtidataservices_app, vendor_default_prop)
+set_prop(qtidataservices_app, exported_radio_prop)
+
+# Find the vendor.qti.hardware.data.iwlan::IIWlan HIDL service
+# And grant binder access to the host (`rild`, `cnd`)
+hal_client_domain(qtidataservices_app, hal_telephony)
+allow qtidataservices_app radio_service:service_manager find;
+
+allow qtidataservices_app self:qipcrtr_socket create_socket_perms_no_ioctl;
+allow qtidataservices_app radio_data_file:dir r_dir_perms;
```

Comment:
Haven't seen any denials but without these two lines I could not make and receive calls but mobile data/service/SMS work.

Reply:
What device is this being tested on? On a SIM/carrier with IMS/VoLTE/whatever-magic support?

Reply:
As mentioned in the PR body, it is the Xperia 5 (kumano/bahamut/j8210).

It is indeed IMS not working (I usually set LTE only and was getting `Connection problem or invalid MMI code`), I have filed https://github.com/sonyxperiadev/device-sony-common/pull/857 in the meantime in a vague attempt to start fixing it :D

Reply:
That is much appreciated. We had a few contributors complain about IMS not working but no work/fixes have come to fruition yet, until these PRs :slightly_smiling_face: 

Note that there is a tricky bit with some apks wanting to run as the `radio` user, which requires them to be re-signed with the same keys as the system (otherwise we could have simply granted all sony-signed apps "platform" permissions with https://github.com/MarijnS95/device-sony-sepolicy/commit/d3061a881077a5a3faabcb2cd13d904e914c1f6a / https://github.com/MarijnS95/device-sony-sepolicy/commit/312123776fb41b2402c943f1d76177c3a21ebbba). I don't know if this only applies to the stack necessary to run VoLTE, but it might hamper IMS as well.